virtool/app.py

```diff
@@ -89,14 +91,15 @@ def create_app(config: Config):
         [
             startup_version,
             startup_events,
+            startup_http_client,
             startup_databases,
             startup_dispatcher,
-            startup_http_client,
             startup_paths,
             startup_routes,
             startup_executors,
             startup_task_runner,
             startup_data,
+            startup_auth,
```

Comment:
Be consistent with `_client`.
```suggestion
            startup_authentication_client,
```